packages/library-base/src/commonMain/kotlin/io/realm/kotlin/serializers/kotlinxserializers/RealmKSerializers.kt

```diff
@@ -0,0 +1,496 @@
+/*
+ * Copyright 2023 Realm Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.realm.kotlin.serializers.kotlinxserializers
```

Comment:
Is there a reason this is in `io.realm.kotlin.serializers.kotlinxserializers` and not just `io.realm.kotlin.serializers`, since the name already says what it is?

Reply:
It is just in case we provide serializers for other serializers but yes, it is already in the serializers names.